### DISCUSSION TITLE
Добрый день, как настроить onboarding, чтобы он не отображался каждый раз при запуске

### DISCUSSION THREAD

**Discussion by GithubGapPoint** [VKCOM/VKUI#4762] [Q&A] [answered]
Хочу сделать отображение онбординг не при каждой запуске, а только при первом 
Делал по этому примеру 
https://codesandbox.io/p/sandbox/vkui-vite-cssm-forked-rztdng?file=%2Fsrc%2FApp.tsx
Помогите пожалуйста, буду благодарен

**Comment by SevereCloud** [comment, marked as answer]
Привет 

> VKUI это библиотека компонентов, а [VKWebAppShowSlidesSheet](https://dev.vk.com/bridge/VKWebAppShowSlidesSheet) относится к [vk-bridge](https://github.com/VKCOM/vk-bridge).

Вопросы стоит спрашивать в соответствующих репозиториях.

---

Необходимо где-то хранить состояние. Как вариант, состояние можно хранить в [VKWebAppStorageSet](https://dev.vk.com/bridge/VKWebAppStorageSet)

Пример https://codesandbox.io/p/sandbox/vkwebappshowslidessheet-forked-50v0td
